R/layer.R

```diff
@@ -437,14 +438,14 @@ Layer <- ggproto("Layer", NULL,
     self$position$compute_layer(data, params, layout)
   },
 
-  compute_geom_2 = function(self, data) {
+  compute_geom_2 = function(self, data, theme) {
```

Comment:
It seems this is one of the common errors in [the revdep check](https://github.com/teunbrand/ggplot2/blob/theme_geom_defaults/revdep/problems.md).

```
Error in compute_geom_2(..., self = self) : 
  argument "theme" is missing, with no default
Calls: <Anonymous> ... use_defaults -> eval_from_theme -> %||% -> calc_element
Execution halted
```

It seems many package use `compute_geom_2(d)` directly. So, probably `theme` argument of `compute_geom_2()` should have the default value of `NULL`.

https://github.com/search?q=org%3Acran+compute_geom_2&type=code